guides/common/modules/proc_performing-an-online-backup.adoc

```diff
@@ -1,21 +1,17 @@
 [id="Performing_an_Online_Backup_{context}"]
 = Performing an online backup
 
-Perform an online backup only for debugging purposes.
-
 ifdef::katello,orcharhino,satellite[]
 .Risks associated with online backups
-When performing an online backup, if there are procedures affecting the Pulp database, the Pulp part of the backup procedure repeats until it is no longer being altered.
-Because the backup of the Pulp database is the most time consuming part of backing up {Project}, if you make a change that alters the Pulp database during this time, the backup procedure keeps restarting.
+When you perform an online backup, most {Project} services remain running and usable.
+Background workers are shut down to ensure consistent backup and operations that require these workers remain in pending state.
```

Comment:
```suggestion
Background workers are shut down to ensure consistent backups.
Operations that require background workers remain in pending state.
```

suggestion (non-blocking): split sentence. I also think we should either say "consistent backups" or "a consistent backup".